pkg/storageincentives/agent.go

```diff
@@ -119,8 +119,14 @@ func (a *Agent) start(blockTime time.Duration, blocksPerRound, blocksPerPhase ui
 		mtx.Unlock()
 
 		if round-1 == sampleRound { // the sample has to come from previous round to be able to commit it
+			a.metrics.CommitPhase.Inc()
```

Comment:
For the phases, we just need a counter and then an error counter,
we do not need this AllPhaseEvents or AllContractCalls